generated/nidaqmx/nidaqmx_service.h

```diff
@@ -68,9 +68,18 @@ ::grpc::Status MonikerWriteBinaryI16(void* data, google::protobuf::Arena& arena,
 ::grpc::Status MonikerWriteBinaryI32(void* data, google::protobuf::Arena& arena, google::protobuf::Any& packedData);
 ::grpc::Status MonikerWriteBinaryU16(void* data, google::protobuf::Arena& arena, google::protobuf::Any& packedData);
 ::grpc::Status MonikerWriteBinaryU32(void* data, google::protobuf::Arena& arena, google::protobuf::Any& packedData);
+::grpc::Status MonikerWriteCtrFreq(void* data, google::protobuf::Arena& arena, google::protobuf::Any& packedData);
```

Comment:
The builds are failing because generated files are being changed when you compile. That's probably just that `test` at the top of cpp files. When you merge from latest main, ensure that all builds pass before completing the PR.